src/kritter/gtabularclient.py

```diff
@@ -10,19 +10,19 @@ class GtabularClient(KtabularClient):
 
     def __init__(self, gcloud):
         self.gcloud = gcloud
-        self.tabScopes = ['https://www.googleapis.com/auth/drive']
-        self.tabCreds = self.gcloud.creds()
-        self.tabClient = gspread.authorize(self.tabCreds)
+        self.tab_creds = self.gcloud.creds()
+        self.tab_client = gspread.authorize(self.tabCreds)
         
     '''
     Creates a google sheet with the requested data 
     '''
-    def createGS(self, fileName, data):
-        self.sheet = self.tabClient.create(fileName)
-        sh = self.tabClient.open(fileName) # opens the created worksheet
+    def create(self, fileName, data):
+        self.sheet = self.tab_client.create(fileName)
```

Comment:
The problem is self.sheet -- it restricts you from being able to deal with more than 1 sheet at a time.   If you return self.sheet (but just return sheet), the user could pass it back when it wants to get_url (for example).   (You need to add sheet arg to get_url.) Does that make sense?  Get rid of self.sheet....